packages/artifact/src/internal/artifact-client.ts

```diff
@@ -240,6 +240,7 @@ Note: The size of downloaded zips can differ significantly from the reported siz
     while (downloadedArtifacts < artifacts.count) {
       const currentArtifactToDownload = artifacts.value[downloadedArtifacts]
       downloadedArtifacts += 1
+      core.info(`starting download of artifact ${currentArtifactToDownload.name} : ${downloadedArtifacts}/${artifacts.count}`)
```

Comment:
Some extra logging that won't hurt. Looks like this: 

![image](https://user-images.githubusercontent.com/16109154/144939040-983c15b4-fbc3-4e7a-b509-dcf849a9dc6a.png)

Reply:
was just gonna leave a comment about this, but it's pretty useful 👍